### PR TITLE
#170947600 Configuring code climate and coveralls

### PR DESCRIPTION
**Pull request by EugeneMunya**
#### What does this PR do?
- display code coverage and climate badges on project READMe.md file
#### Description of Task to be completed?
- add code coverage and climate badges on READMe.md file
#### How should this be manually tested?
- go to the [READMe.md](https://github.com/andela/knights-bn-frontend/tree/ch-test-coverage-170947600) file
#### What are the relevant pivotal tracker stories?
[#170947600](https://www.pivotaltracker.com/story/show/170947600)
#### Screenshots (if appropriate)
Screen Shot 2020-04-21 at 10 36 55

### CODE REVIEW

**Review by Niyonsengaeric — APPROVED** [review]

**Comment by T2Wil** [comment]
Hey @EugeneMunya , here are a couple of things you should fix:
- combine the two commits into one.
- the `Readme.md` link provided in the PR description is redirecting to the develop branch and there are no badges.
- Fix the pivotal tracker story link in the PR description

**Comment by T2Wil** [comment]
@EugeneMunya , From the PR title there is a grammatical error. 
configuringing => configuring.
Consider installing Grammarly extension in your browser to prevent this in the future.

**Comment by Kagorora** [comment]
@EugeneMunya You have two commits message, consider squashing

**Comment by EugeneMunya** [comment]
> Hey @EugeneMunya , here are a couple of things you should fix:
> 
> * combine the two commits into one.
> * the `Readme.md` link provided in the PR description is redirecting to the develop branch and there are no badges.
> * Fix the pivotal tracker story link in the PR description

I did correction you can check now

**Comment by EugeneMunya** [comment]
> @EugeneMunya You have two commits message, consider squashing

I resolved the issue check again

File: jest.config.js [hunk 1]
```diff
@@ -10,6 +10,6 @@ module.exports = {
   ],
   transform: {
     '^.+\\.[t|j]sx?$': 'babel-jest',
-    '^.+\\.scss$': 'sass-jest',
+    '^.+\\.scss$': '<rootDir>/node_modules/jest-css-modules-transform',
```
Comment (T2Wil) [inline jest.config.js:13]: why did you change this transform?

**Review by Kagorora — APPROVED** [review]
I have seen his PR and everything works

File: jest.config.js (hunk 1 above)
Comment (EugeneMunya) [reply jest.config.js:13]: I was asked to install `jest-CSS-modules-transform` module in order to run  jest test

**Comment by T2Wil** [comment]
Your `Readme.md` link in the PR should be linked to your branch : 
`https://github.com/andela/knights-bn-frontend/tree/ch-test-coverage-170947600` 
not to develop

**Comment by EugeneMunya** [comment]
> Your `Readme.md` link in the PR should be linked to your branch :
> `https://github.com/andela/knights-bn-frontend/tree/ch-test-coverage-170947600`
> not to develop

I have changed it

**Review by T2Wil — APPROVED** [review]

**Review by kagabof — APPROVED** [review]
Well done.
